client/apps/edit2/components/content/section_controls/index.jsx

```diff
@@ -34,143 +44,122 @@ export class SectionControls extends Component {
     // Add extra space for channels with Yoast
-    return this.props.channel.type === 'partner' ? 55 : 95
+    return type === 'partner' ? 55 : 95
   }
 
-  getPositionBottom () {
-    if (this.state.insideComponent && !this.props.isHero) {
-      return window.innerHeight - $(this.refs.controls).height() - this.getHeaderSize()
-    } else {
-      return '100%'
-    }
-  }
+  getPositionBottom = () => {
+    if (this.controls) {
+      const { insideComponent } = this.state
+      const { isHero } = this.props
 
-  isScrollingOver ($section) {
-    const scrollPlusHeader = window.scrollY + this.getHeaderSize()
-    const offsetMinusControls = $section.offset().top - $(this.refs.controls).height()
-    return scrollPlusHeader > offsetMinusControls
-  }
-
-  isScrolledPast ($section) {
-    const scrollPlusControls = window.scrollY + $(this.refs.controls).height()
-    const scrollPlusSection = $section.offset().top + $section.height()
-    return scrollPlusControls > scrollPlusSection
-  }
+      const controlsHeight = $(this.controls).height()
+      const windowHeight = window.innerHeight
+      const headerHeight = this.getHeaderHeight()
 
-  insideComponent = () => {
-    let insideComponent = false
-    const $section = $(this.refs.controls) && $(this.refs.controls).closest('section')
+      const stickyBottom = windowHeight - controlsHeight - headerHeight
 
-    if ($section && !this.props.isHero) {
-      if ((this.isScrollingOver($section) && !this.isScrolledPast($section)) ||
-       (this.props.isHero && !this.isScrolledPast($section))) {
-        insideComponent = true
+      if (insideComponent && !isHero) {
+        return stickyBottom + 'px'
+      } else {
+        return '100%'
       }
     }
-    return insideComponent
   }
 
-  changeLayout (layout) {
-    if (layout === 'fillwidth' && this.sectionIsImage() &&
-     this.props.section.get('images').length > 1) {
-      return this.props.disabledAlert()
-    }
-    if (this.props.section.get('type') === 'image_set') {
-      this.props.section.set('type', 'image_collection')
-      this.forceUpdate()
-    }
-    this.props.section.set({ layout })
-    this.props.onChange && this.props.onChange()
-  }
+  isScrollingOver = ($section) => {
+    if (this.controls) {
+      const controlsHeight = $(this.controls).height()
+      const offsetTop = $section.offset().top
 
-  toggleImageSet = () => {
-    if (this.props.section.get('type') === 'image_collection') {
-      this.props.section.set({type: 'image_set', layout: 'mini'})
-      this.forceUpdate()
+      const scrollPlusHeader = window.scrollY + this.getHeaderHeight()
+      const offsetMinusControls = offsetTop - controlsHeight
+
+      return scrollPlusHeader > offsetMinusControls
     }
-    this.props.onChange && this.props.onChange()
   }
 
-  hasImageSet () {
-    const { type } = this.props.channel
-    return this.sectionIsImage() && type === 'editorial' || type === 'team'
-  }
+  isScrolledPast = ($section) => {
```

Comment:
Its common to prefix booleans with `is`, but can also use other forms such as `hasScrolledPast` if it reads a bit better. Either way; `is` is definitely a noticeable flag.